.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "openzwave-sys/open-zwave"]
+	path = openzwave-sys/open-zwave
+	url = https://github.com/OpenZWave/open-zwave.git
```

Comment:
FYI: cargo doesn't seem to update submodules (at least not for local builds).
I had to do git submodule init && git submodule update to get the build to work.